src/Analyzer.TemplateProcessor/Analyzer.TemplateProcessor.csproj

```diff
@@ -7,9 +7,9 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Azure.Deployments.Core" Version="1.0.635" />
-    <PackageReference Include="Azure.Deployments.Expression" Version="1.0.635" />
-    <PackageReference Include="Azure.Deployments.Templates" Version="1.0.635" />
+    <PackageReference Include="Azure.Deployments.Core" Version="1.0.694" />
```

Comment:
Does the lastest version 1.0.765 also work here?

Reply:
1.0.750 is latest for me